rosidl_generator_py/msg/Strings.msg

```diff
@@ -1,5 +1,9 @@
 string empty_string
 string def_string "Hello world!"
+string def_string_delimiter 'Hello"world!'
+string def_string_delimiter2 "Hello'world!"
+string def_string_delimiter3 'Hello'world!'
+string def_string_delimiter4 "Hello"world!"
```

Comment:
I updated the referenced comment to put the snippet in a code block since some of the backslashes weren't visible. From a generic parsing point of view it can't be `'foo'bar'` imo.